sourcecode/apis/contentauthor/resources/assets/react/components/QuestionContentType/components/QuestionCard/QuestionCardLayout.js

```diff
@@ -41,10 +41,13 @@ function QuestionCardLayout(props) {
                     showAddAnswer={showAddAnswerButton}
                 />
                 {typeof deleteCard === 'function' && (
-                    <AddIcon
+                    <button
                         className="deleteButton"
                         onClick={deleteCard}
-                    />
+                        aria-label="Delete button"
```

Comment:
`aria-label` should describe what the button does, not what the button is. This must also be translatable.